templates/java/mobile/app/build.gradle.ftl

```diff
@@ -1,44 +1,42 @@
-buildscript {
-    repositories {
-        jcenter()
-        <#if configs.dependencies.fabrickey??>
-            maven { url 'https://maven.fabric.io/public' }
-        </#if>
-    }
-    dependencies {
-        <#if configs.dependencies.fabrickey??>
-        classpath 'io.fabric.tools:gradle:1.22.1'
-        </#if>
-        <#if configs.qualityVerifier??>
-        classpath 'pt.simdea.verifier:verifier:3.5.0'
-        </#if>
-    }
-}
-
+apply plugin: 'com.android.application'
 <#if configs.dependencies.fabrickey??>
-repositories {
-    maven { url 'https://maven.fabric.io/public' }
-}
-
+apply plugin: 'io.fabric'
+</#if>
+apply from: "$project.rootDir/tools/git-version.gradle"
+<#if configs.qualityVerifier??>
+apply plugin: 'pt.simdea.verifier'
 </#if>
-apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion ${configs.targetSdkApi}
-    buildToolsVersion '${configs.buildTools}'
+
+    compileSdkVersion project.COMPILE_SDK_VERSION.toInteger()
+    buildToolsVersion project.BUILD_TOOLS_VERSION
 
     defaultConfig {
-        minSdkVersion ${configs.minimumSdkApi}
-        targetSdkVersion ${configs.targetSdkApi}
+        applicationId project.APP_ID
+        minSdkVersion project.MIN_SDK_VERSION.toInteger()
+        targetSdkVersion project.TARGET_SDK_VERSION.toInteger()
         <#if configs.dependencies.retrofit??>
         	buildConfigField ("long", "API_TIMEOUT", "${configs.dependencies.retrofit.timeout}")
         </#if>
     }
 
+    packagingOptions {
```

Comment:
@Andrer757 is this necessary?

Reply:
Yes, this option is because many libs use that unneeded files and build fails because that file already exists from another lib.